node/index.ts

```diff
@@ -909,3 +957,81 @@ export class UnidentifiedSenderMessageContent {
     );
   }
 }
+
+/*
+export interface SessionStore {
+  storeSession(address: ProtocolAddress, session: SessionRecord): void;
+  loadSession(address: ProtocolAddress): SessionRecord | null;
+}
+
+export const enum Direction {
+  Sending,
+  Receiving,
+}
+
+export interface IdentityKeyStore {
+  getIdentityKeyPair(): IdentityKeyPair;
+  getLocalRegistrationId(): number;
+  saveIdentity(address: ProtocolAddress, identity: PublicKey): void;
+  isTrustedIdentity(
+    address: ProtocolAddress,
+    key: PublicKey,
+    dir: Direction
+  ): boolean;
+  getIdentity(address: ProtocolAddress): PublicKey | null;
+}
+
+export interface PreKeyStore {
+  getPreKey(id: number): PreKeyRecord;
+  savePreKey(id: number, record: PreKeyRecord): void;
+  removePreKey(id: number): void;
+}
+
+export interface SignedPreKeyStore {
+  getSignedPreKey(id: number): SignedPreKeyRecord;
+  saveSignedPreKey(id: number, record: SignedPreKeyRecord): void;
+}
+*/
+
+export abstract class SenderKeyStore {
+  _saveSenderKey(
+    name: SignalClient.SenderKeyName,
+    record: SignalClient.SenderKeyRecord
+  ): Promise<void> {
```

Comment:
`async` and Promise are interchangeable in JavaScript, so you should be able to write these as `async` functions instead.